semanticdb/cli/src/main/scala/scala/meta/cli/Reporter.scala

```diff
@@ -11,13 +11,26 @@ final class Reporter private (val out: PrintStream, val err: PrintStream) {
     copy(out = out)
   }
 
+  def silenceOut(): Reporter = {
```

Comment:
Nitpick: I find it weird to use the builder pattern with methods that are not named "withX". I would have named this method `withSilentOut`

Reply:
Will fix.